apps/console/src/features/core/configs/routes.tsx

```diff
@@ -1680,6 +1680,20 @@ export const getAppViewRoutes = (useExtendedRoutes: boolean = false): RouteInter
                 path: AppConstants.getPaths().get("USERSTORES"),
                 protected: true,
                 showOnSidePanel: true
+            },
+            {
+                category: "extensions:manage.sidePanel.categories.userManagement",
+                component: lazy(() => import("../../provisioning/pages/outbound-provisioning-settings")),
+                exact: true,
+                icon: {
+                    icon: <UserGroupIcon className="icon" fill="black" />
```

Comment:
aren't we using the same icon for groups?

Reply:
Updated with b2b26abeb4ea42fd52d73cd016be02dc68af453f